client/src/components/rows/VulnerabilityViewRow.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+interface VulnerabilityViewRowProps {
```

Comment:
would advise to make a common rowprops interface to make vulnerabilityviewrow and componentviewrow enums or "strategies" like Zoe did with the query component - but overall good job!